src/main/java/com/cupid/jikting/like/service/LikeService.java

```diff
@@ -24,12 +24,17 @@ public List<LikeResponse> getAllReceivedLike(Long memberProfileId) {
         return getTeamMemberById(memberProfileId)
                 .getReceivedTeamLikes()
                 .stream()
-                .map(LikeResponse::from)
+                .map(LikeResponse::fromReceivedTeamLike)
                 .collect(Collectors.toList());
     }
 
-    public List<LikeResponse> getAllSentLike() {
-        return null;
+    @Transactional(readOnly=true)
```

Comment:
p1. `getAllReceivedLike()`도 아래처럼 같이 연산자 앞뒤에 공백 적용해주세요!

```suggestion
    @Transactional(readOnly = true)
```